File: official/docs/csharp/current/claims/cancel.cs
using System;
using System.Collections.Generic;
using System.Threading.Tasks;
using Newtonsoft.Json;
using EasyPost;
using EasyPost.Models.API;

namespace EasyPostExamples
{
    public class Examples
    {
        public static async Task Main()
        {
            var client = new EasyPost.Client(new EasyPost.ClientConfiguration("EASYPOST_API_KEY"));

            Claim claim = await client.Claim.Cancel("clm_...");

            Console.WriteLine(JsonConvert.SerializeObject(claim, Formatting.Indented));
        }
    }
}

File: official/docs/golang/current/claims/cancel.go
package example

import (
	"fmt"

	"github.com/EasyPost/easypost-go/v5"
)

func cancel() {
	client := easypost.New("EASYPOST_API_KEY")

	claim, _ := client.CancelClaim("clm_...")

	fmt.Println(claim)
}
